content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Use a SOCKS5 Proxy to Access the Kubernetes API
+content_type: task
+weight: 42
+min-kubernetes-server-version: v1.24.0
```

Comment:
should this be `min-kubernetes-version: 1.24`? I'm not sure.

Reply:
you're right, corrected.